app/services/revocation_registry.py

```diff
@@ -425,3 +509,48 @@ async def get_credential_definition_id_from_exchange_id(
         "Successfully obtained cred definition id from the cred exchange id."
     )
     return credential_definition_id
+
+
+async def validate_rev_reg_ids(controller: AcaPyClient, request: dict):
+    """
+        Validate revocation registry ids
+
+    Args:
+        controller (AcaPyClient): aca-py client
+        request (dict): A map of revocation registry ids to a list of credential revocation ids
```

Comment:
`request: dict` can be better named: `revocation_registry_credential_map: Dict[str, List[str]]` 
(seems that's what it must be)